src/panels/config/entities/ha-config-entities.ts

```diff
@@ -363,6 +366,10 @@ export class HaConfigEntities extends SubscribeMixin(LitElement) {
           continue;
         }
 
+        if (!showAreaConfigured && areaId) {
```

Comment:
I am still not sure if this should be `area` or `areaId`.

Reply:
`areaId` is fine here